src/styles/global.css

```diff
@@ -17,3 +17,7 @@ a:hover {
 .content ul {
   @apply my-6;
 }
+
+.check-box-1:checked ~ .check-1 {
+  @apply text-opacity-100;
+}
```

Comment:
Remove these lines as we do not need it anymore. Also remove classNames check-box and check-1 from the checkbox as it doesnt use it anymore.

Reply:
Done